paymentsheet/src/main/java/com/stripe/android/link/ui/updatecard/UpdateCardScreen.kt

```diff
@@ -28,18 +27,20 @@ import com.stripe.android.paymentsheet.ui.CardDetailsEditUI
 import com.stripe.android.paymentsheet.ui.DefaultEditCardDetailsInteractor
 import com.stripe.android.paymentsheet.ui.EditCardDetailsInteractor
 import com.stripe.android.paymentsheet.ui.EditCardPayload
-import com.stripe.android.R as StripeR
 import com.stripe.android.uicore.utils.collectAsState
+import com.stripe.android.R as StripeR
 
 @Composable
 internal fun UpdateCardScreen(viewModel: UpdateCardScreenViewModel) {
     val state by viewModel.state.collectAsState()
-    UpdateCardScreenBody(
-        interactor = viewModel.interactor,
-        isDefault = state?.paymentDetails?.isDefault == true,
-        onUpdateClicked = viewModel::onUpdateClicked,
-        onCancelClicked = viewModel::onCancelClicked,
-    )
+    viewModel.interactor?.let {
```

Comment:
Could this cause any rendering issues when this screen opens?

Reply:
this should never happen as it gets initialized on the viewmodel constructor, but just in case: https://github.com/stripe/stripe-android/pull/10715/commits/8fe4434e5b46e4dd6c02529307d904b8646c19d1